examples/src/main/java/com/mapbox/navigation/examples/core/SimpleMapboxNavigationKt.kt

```diff
@@ -247,12 +247,9 @@ class SimpleMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback,
         dismissLayout.setOnClickListener {
             fasterRouteSelectionTimer.onFinish()
         }
-        acceptLayout.setOnClickListener {
-            fasterRoute?.let {
-                mapboxNavigation.setRoutes(mapboxNavigation.getRoutes().toMutableList().apply {
-                    removeAt(0)
-                    add(0, it)
-                })
+        acceptLayout.setOnClickListener { _ ->
+            fasterRoutes.takeIf { it.isNotEmpty() }?.let { newRoutes ->
```

Comment:
nit: `fasterRoutes.takeIf { it.isNotEmpty() }?.let { newRoutes ->`

better book-keeping. wonder if core can/should help with this

Reply:
Look at this `TODO` / question 😅 https://github.com/mapbox/mapbox-navigation-android/blob/3fd77323074d64217606f85291cd38b9f51297d5/libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt#L113-L114

Reply:
yeah we rely on list order in a lot of places now. another option is to add a function that does something like this

https://github.com/mapbox/mapbox-navigation-android/blob/80d5359f2abb6d82bd5613b077bdbc3902b63143/examples/src/main/java/com/mapbox/navigation/examples/core/FasterRouteActivity.kt#L241-L244

``` kotlin
MapboxNavigation

    fun setRoutes(routes: List<DirectionsRoute>) {
        directionsSession.routes = routes
    }
    fun setRoute(route: DirectionsRoute) {
        setRoutes(mapboxNavigation.getRoutes().toMutableList().apply {
            removeAt(0)
            add(0, it)
        })
    }